src/frontend/src/pages/FinancePage/FinanceComponents/GenerateReceiptsModal.tsx

```diff
@@ -12,18 +12,22 @@ import { ReimbursementRequest } from 'shared';
 
 const schema = yup.object().shape({
   startDate: yup.date().required('Start Date is required'),
-  endDate: yup.date().min(yup.ref('startDate'), `end date can't be before start date`).required('End Date is required')
+  endDate: yup.date().min(yup.ref('startDate'), `end date can't be before start date`).required('End Date is required'),
+  refundSource: yup.string()
```

Comment:
this should be required since the options are both, cash, and budget. For start date/ end date you could make them optional, then just return all before the end date if start date isn't filled out, all after the start date if end date isn't filled out and all of the receipts if neither are filled out